lib/matplotlib/widgets.py

```diff
@@ -1627,6 +1785,9 @@ def __init__(self, ax, labels, active=0, activecolor=None, *,
         _api.check_isinstance((dict, None), label_props=label_props,
                               radio_props=radio_props)
 
+        labels = list(labels)
+        n_labels = len(labels)
+
```

Comment:
```suggestion
```
Conversion to list is not necessary. Also before, we required that `labels` is iterable and has a `__len__`.
`n_labels` is only used once in the function -> no benefit in declaring a separate variable; keep inline